ai.finto.fi/static/js/annif.js

```diff
@@ -435,23 +436,44 @@ mainApp.component('limit-input', {
 })
 
 mainApp.component('language-select', {
-  props: ['modelValue'], // modelValue: selected language
+  props: ['modelValue', 'selected_project'],
   emits: ['update:modelValue'],
+  computed: {
+    vocabularyId() {
+      // TODO: This is a hack. We should expose the vocabulary id from Annif API.
+      // Why project is here the project_id string?
+      // Assume vocabulary id is a prefix
+      if (this.selected_project) {
+        return this.selected_project.split("-")[0];
```

Comment:
I could not use camelCased name `selectedProject` in this method/component, but only `selected_project` worked.